Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic_README.md

```diff
@@ -0,0 +1,71 @@
+This playbook performs an investigation on a specific user, using queries and logs from Siem, Identities management systems, XDR, and Firewalls.
```

Comment:
SIEM and not Siem
Identity and not Identities

Reply:
Done